### ISSUE TITLE
CUSA17416 - Persona 5 Royal

### ISSUE THREAD

**Issue by StevenMiller123** [shadps4-emu/shadps4-game-compatibility#55]
### Game Name

Persona 5 Royal

### Game code

CUSA17416

### Game version

v1.00

### Used emulator's version (only released versions are acceptable)

v0.1.0

### Current status

Nothing

### Error

Crashes after calling various sceFiber functions. 
[Core] <Error> stubs.cpp:CommonStub:42: Stub: _sceFiberInitializeImpl (nid: hVYD7Ou2pCQ) called, returning zero to 0x9000c3013
[Core] <Error> stubs.cpp:CommonStub:42: Stub: sceFiberRun (nid: a0LLrZWac0M) called, returning zero to 0x9000a64ab
[Core] <Error> stubs.cpp:CommonStub:42: Stub: sceFiberGetSelf (nid: p+zLIOg27zU) called, returning zero to 0x9000c3257

### Description

The game crashes instantly.

### Log File

[Persona 5 Royal Log.txt](https://github.com/user-attachments/files/16075564/Persona.5.Royal.Log.txt)

**Comment by StevenMiller123** [comment]
On emulator v0.2.0, Persona 5 Royal no longer crashes. The game can reach ingame, however it has severe performance issues making it difficult to test (I've provided a video for reference). 

https://github.com/user-attachments/assets/6de9d93d-f2ac-4c74-8626-a622e53425f2

The game should probably be marked as Ingame, although reaching that point for screenshots is rather time consuming. 
Log: [Persona 5 Royal log.txt](https://github.com/user-attachments/files/16631819/Persona.5.Royal.log.txt)

**Comment by StevenMiller123** [comment]
On emulator v0.3.0, the game's Windows-exclusive performance issues are resolved. The game hangs on menus due to needing `sceVideodec2QueryDecoderMemoryInfo`.

This game's status should be changed to menus.
[shad_log.txt](https://github.com/user-attachments/files/17123293/shad_log.txt)

**Comment by Cyclizar** [comment]
Just tested and waited on the loading screen for like 2 hours, I'll test tomorrow with a save file. log consists of:
[Lib.VideoOut] <Info> video_out.cpp:sceVideoOutSetFlipRate:109: called
[Tty] <Info> libkernel.cpp:ps4__write:211:  # CRI # E2008090501:Failed to get compressed data size.
[Lib.VideoOut] <Info> video_out.cpp:sceVideoOutIsFlipPending:115: called
[Tty] <Info> libkernel.cpp:ps4__write:211:  # CRI # E2008090501:Failed to get compressed data size.
[Tty] <Info> libkernel.cpp:ps4__write:211:  # CRI # E2008090501:Failed to get compressed data size.

id send the full log but its 100mb lmao

@StevenMiller123 were you able to get past the loading screen?

**Comment by StevenMiller123** [comment]
@Cyclizar There's no point in waiting, as the game is hanging due to the lack of support for those sceVideodec2 functions. Back in some 0.1.1 WIP builds, there was a bug with the emulator that got the game to skip these function calls. This issue eventually got fixed, though because of the performance issues on Windows, I never noticed the game "regressing" until recent tests. 

TLDR: Current builds can't go ingame, no matter how long you wait.

**Comment by Cyclizar** [comment]
ahh i see, could a patch be created to recreate the bug from v0.1.1 do you think? @StevenMiller123

**Comment by Yentis** [comment]
Gets to menus (with graphical glitches) on 0.4.0, black screen when starting new game.
![shadPS4_FDcgJtby6z](https://github.com/user-attachments/assets/7b68f773-626a-4d0c-9a72-8862acb30e42)

**Comment by Yentis** [comment]
Gets ingame with graphical glitches on 0.5.0
![shadPS4_rIaMERWX4u](https://github.com/user-attachments/assets/04e3318b-3aa8-4d05-9410-45c7454b90d6)
[shad_log.txt](https://github.com/user-attachments/files/18247365/shad_log.txt)

**Comment by Finny7667** [comment]
Still graphical Glitches in the 6.1 WIP
Framerate is playable
![Image](https://github.com/user-attachments/assets/42d9126b-28fc-4793-b044-a07fdf05ec8a)

![Image](https://github.com/user-attachments/assets/c9334d14-cd7f-4fe2-b500-ab5f2b440494)
